storage/books.json

```diff
@@ -0,0 +1 @@
+[{"title":"Don quijote","author":"Cervantes"}]
```

Comment:
- [ ] [OPTIONAL] Great job team 👍🏽 
 The data is been saved successfully into the JSON file. So, please remove the data currently in the JSON file so fresh data can be saved. I removed it before testing and it worked perfectly 👍🏽 
 
<img width="1086" alt="Screenshot 2023-04-19 at 17 22 09" src="https://user-images.githubusercontent.com/58692590/233138947-884fd66a-8b70-4d7d-b855-ed4b318d32d0.png">

Let's implement this for the `persons.json` and the `rentals.json` as well
We can fix this 💪🏽